NetSDK/Services/Impressions/Classes/SelfUpdatingTreatmentLog.cs

```diff
@@ -22,7 +22,7 @@ public class SelfUpdatingTreatmentLog: ITreatmentLog
 
         protected static readonly ILog Logger = LogManager.GetLogger(typeof(SelfUpdatingTreatmentLog));
 
-        public SelfUpdatingTreatmentLog(ITreatmentSdkApiClient apiClient, int interval = 180, BlockingQueue<KeyImpression> queue = null, int maximumNumberOfKeysToCache = -1)
+        public SelfUpdatingTreatmentLog(ITreatmentSdkApiClient apiClient, int interval = 60, BlockingQueue<KeyImpression> queue = null, int maximumNumberOfKeysToCache = -1)
```

Comment:
We already defined the defaults into the object, there is no need for defaults here, please use always the configurations.